bobcat/bb-cumber/src/main/java/com/cognifide/qa/bb/cumber/Bobcumber.java

```diff
@@ -57,19 +56,19 @@
 
   private static final Logger LOG = LoggerFactory.getLogger(Bobcumber.class);
 
-  private static final Map<String, Set<String>> ADDED_FEATURES = new HashMap<>();
+  private final Properties properties = PropertyUtils.gatherProperties();
 
-  private static final String COLON = ":";
+  private final double maxFailedTestPercentage = Double.parseDouble(properties.getProperty(ConfigKeys.BOBCAT_REPORT_STATISTICS_PERCENTAGE));
 
-  private static final String FEATURE = "feature";
+  private boolean storeFailedResults;
 
-  private static final String RERUN_FAILED_TESTS_CLASS_NAME = "com.cognifide.qa.bobcumber.RerunFailedTests";
+  private boolean isItFailedTestsRerun;
 
-  private boolean storeFailedResults;
+  private File featureFile;
 
-  private boolean noTestsToRerun;
+  private File statisticsFile;
 
-  private File file;
+  private StatisticsHelper statisticsHelper = new StatisticsHelper();
```

Comment:
could be final and please be consistent with initialization - initialize all fields in the constructor